mypy/suggestions.py

```diff
@@ -179,7 +221,7 @@ def __init__(self, fgmanager: FineGrainedBuildManager,
         if no_any:
             self.flex_any = 1.0
 
-        self.max_guesses = 16
+        self.max_guesses = 64
```

Comment:
An unrelated note: maybe we should make this configurable?

Reply:
Yeah I'll do that in a follow-up soon